solidity/deploy/34_deploy_wallet_coordinator.ts

```diff
@@ -0,0 +1,43 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, ethers, getNamedAccounts, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const Bridge = await deployments.get("Bridge")
+
+  const [walletCoordinator, proxyDeployment] =
+    await helpers.upgrades.deployProxy("WalletCoordinator", {
+      contractName: "WalletCoordinator",
+      initializerArgs: [Bridge.address],
+      factoryOpts: {
+        signer: await ethers.getSigner(deployer),
+      },
+      proxyOpts: {
+        kind: "transparent",
+      },
+    })
```

Comment:
Non-blocking this PR but let's verify the behavior: 

OpenZeppelin deploys only one `ProxyAdmin` contract for the given network. All transparent proxy contracts are owned by it. Since `Bridge` contract is already deployed on mainnet with its own `ProxyAdmin`, we must ensure the underlying mechanism detects the existing `ProxyAdmin` properly.